docs/sql-reference/sql-statements/data-definition/CREATE TABLE.md

```diff
@@ -808,6 +827,29 @@ PROPERTIES(
 );
 ```
 
+### Create a table where multiple partitions created at a time, and an integer type column is specified as partitining column
```

Comment:
1. multiple partitions are created    2. partitining --- spelling error  the partitioning column